foca/models/config.py

```diff
@@ -528,15 +541,22 @@ class MongoConfig(FOCABaseConfig):
 
     Example:
         >>> MongoConfig(
-        ...     host="mongodb",
-        ...     port=27017,
+        ...     host="mongodb_host",
+        ...     port=27018,
         ... )
-        MongoConfig(host='mongodb', port=27017, dbs=None)
+        MongoConfig(host='mongodb_host', port=27018, dbs=None)
     """
-    host: str = "mongodb"
-    port: int = 27017
+    username: Optional[str] = Field(default=None)
+    password: Optional[str] = Field(default=None)
+    host: str = Field(default="mongodb")
```

Comment:
This very quickly becomes much more verbose. I'm really not convinced. I think we should really just do this for secrets.